### DISCUSSION TITLE
Was wire:poll="foo" left out intentionally in V3?

### DISCUSSION THREAD

**Discussion by skeemer** [livewire/livewire#6205] [1. Help] [answered]
The advantage I saw for this was to be able to use conditional re-render skipping to conserve resources.

**Comment by PhiloNL** [comment]
wire:poll is still there: https://livewire.laravel.com/docs/polling

**Reply by skeemer** [reply]
Not with ="some_action_to_call".

**Reply by PerryvanderMeer** [reply, marked as answer]
You still can, but it wasn't documented.
https://github.com/livewire/livewire/pull/6226
